README.md

```diff
@@ -207,7 +207,7 @@ docker push ${image_repo_and_project}/pulsar:$tag
 Apache Pulsar is using [lombok](https://projectlombok.org/) so you have to ensure your IDE setup with
 required plugins.
 
-### Intellij
+### IntellJ
```

Comment:
```suggestion
### IntelliJ
```

Reply:
@Demogorgon314 , Thanks your review，I will update it soon.